src/Exceptions/HttpLocked.php

```diff
@@ -8,4 +8,3 @@ class HttpLocked extends HttpException {
     public const DESCRIPTION = "Means the source or destination resource of a method is locked.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/423";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces